pkg/tests/apis/folder/folders_test.go

```diff
@@ -474,6 +492,36 @@ func doNestedCreateTest(t *testing.T, helper *apis.K8sTestHelper) {
 	require.Equal(t, parentCreate.Result.URL, parent.URL)
 }
 
+func doCreateDuplicateFolderTest(t *testing.T, helper *apis.K8sTestHelper) {
+	client := helper.GetResourceClient(apis.ResourceClientArgs{
+		User: helper.Org1.Admin,
+		GVR:  gvr,
+	})
+
+	payload := `{
+		"title": "Test",
+		"uid": ""
+		}`
+	create := apis.DoRequest(helper, apis.RequestParams{
+		User:   client.Args.User,
+		Method: http.MethodPost,
+		Path:   "/api/folders",
+		Body:   []byte(payload),
+	}, &folder.Folder{})
+	require.NotNil(t, create.Result)
+	parentUID := create.Result.UID
+	require.NotEmpty(t, parentUID)
+
+	create2 := apis.DoRequest(helper, apis.RequestParams{
+		User:   client.Args.User,
+		Method: http.MethodPost,
+		Path:   "/api/folders",
+		Body:   []byte(payload),
+	}, &folder.Folder{})
+	require.NotEmpty(t, create2.Response)
+	require.Equal(t, 409, create2.Response.StatusCode)
```

Comment:
Nitpick, could we use the `http` status constant instead the raw code?

```suggestion
	require.Equal(t, http.StatusConflict, create2.Response.StatusCode)
```